server/admin-function/search-user-by-email.ts

```diff
@@ -0,0 +1,37 @@
+"use server";
+
+import { dbAdmin } from "@/lib/firebase-admin";
+import { DocumentData, QuerySnapshot } from 'firebase-admin/firestore';
+
+// Function to simulate fetching user data
+export const searchUserByEmail = async (domain: string, email: string): Promise<DocumentData | null> => {
+  try {
+
+    console.log('searchUserByEmail:', domain, email)
+    // Check if the logged-in user has an admin role
+
+    if (true) {
```

Comment:
if (true) is completely useless here, what condition did you meant to verify??

console.log('User does not have admin role'); will never be called

Reply:
oh, this isnt supposed to be there